### PR TITLE
Cleans up additional artifacts in uninstall. Closes 3082

### LINKED ISSUE #3082: OpenShift uninstall leaves some configurations behind
From https://bugzilla.redhat.com/show_bug.cgi?id=1412559:

Description of problem:
I'm using the following stanza to finish cleaning up after uninstallation, I think removal some/all of these (perhaps except for docker) should be part of the real uninstall procedure:

	find \
	/var/lib/docker/*
        /etc/sysconfig/atomic-* \
        /etc/pki/ca-trust/source/anchors/openshift* \
        /etc/openvswitch /var/log/openvswitch \
        -maxdepth 1 -print 2> /dev/null | xargs rm -rf

Thanks.

Version-Release number of selected component (if applicable):
openshift-ansible-playbooks-3.3.54-1.git.0.61a1dee.el7.noarch

### PR DESCRIPTION
**Pull request by gregswift**
Closes #3082

### CODE REVIEW

File: playbooks/adhoc/uninstall.yml [hunk 1]
```diff
@@ -151,6 +151,14 @@
       - lbr0
       - vlinuxbr
       - vovsbr
+
+    - name: Remove virtual devices
+      command: nmcli delete device "{{ item }}"
+      failed_when: False
+      with_items:
+      - tun0
+      - docker0
```
Comment (gregswift) [inline playbooks/adhoc/uninstall.yml]: the above ip link delete doesnt work for these

**Comment by gregswift** [comment]
occurs to me that the uninstall is specifically not cleaning up docker for some reason and that reason is probably that y'all expect those things to be ready before.. not sure what to do with

**Comment by michaelgugino** [comment]
@sdodson does this look okay to you?  Looks okay to me.

**Comment by sdodson** [comment]
Yeah, looks fine to me. We're pretty loose with this playbook given its nature.
/lgtm

**Pull request merged**